open-craft/xblock-group-project-v2
# Peer grading stages are marked as completed for TA grader

And some cleanup.

**Testing instructions:**
1. Log in as TA
2. Go to TA grading view (see instructions on almost every PR in this repo).
3. Provide feedback for the group.
4. Observe that stage is marked as completed in PN.
